docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+php_fpm:
+  image: 'docker.zicht.nl/php-fpm:7.2'
```

Comment:
Dit is een prive-repo, hebben wij deze bestanden bij al onze bundles er in zitten?

Reply:
Had jij er in gezet :wink:  Ik heb het geloof ik niet eerder gezien bij bundles